nodes/object_operators/ScClearParent.py

```diff
@@ -0,0 +1,27 @@
+import bpy
+
+from bpy.props import EnumProperty
+from bpy.types import Node
+from .._base.node_base import ScNode
+from .._base.node_operator import ScObjectOperatorNode
+
+class ScClearParent(Node, ScObjectOperatorNode):
+    bl_idname = "ScClearParent"
+    bl_label = "Clear Parent"
+
+    ops_type: EnumProperty(items=[('CLEAR', 'Normal', ''), ('CLEAR_KEEP_TRANSFORM', 'Keep Transform', ''), ('CLEAR_INVERSE', 'Clear Inverse', '')], default='CLEAR', update=ScNode.update_value)
+
+    def init(self, context):
+        super().init(context)
+        self.inputs.new("ScNodeSocketString", "Type").init("ops_type")
```

Comment:
Since this is the only input parameter (apart from object), keep it visible by default for better UX.

Reply:
Fixed.